src/reducers/collectiblesReducer.js

```diff
@@ -82,12 +86,24 @@ const collectiblesReducer = (
   state: CollectiblesReducerState = initialState,
   action: CollectiblesAction,
 ): CollectiblesReducerState => {
+  let accountId;
+
   switch (action.type) {
-    case UPDATE_COLLECTIBLES:
+    case SET_COLLECTIBLES:
       return {
         ...state,
         data: action.payload || {},
       };
+    case SET_ACCOUNT_COLLECTIBLES:
+      ({ accountId } = action.payload);
```

Comment:
Do we need the parenthesis here?

Reply:
Yeah, either we need to kill that eslint rule or keep applying like this 👀